packages/catalog-generator/scripts/build-catalogs.mjs

```diff
@@ -15,13 +15,34 @@ const require = createRequire(import.meta.url);
 const CATALOGS = {
   // https://repo1.maven.org/maven2/org/apache/camel/camel-catalog/
   // https://maven.repository.redhat.com/ga/org/apache/camel/camel-catalog/
-  Main: ['4.9.0', '4.8.2', '4.4.4', '4.4.0.redhat-00046', '4.8.0.redhat-00017'],
+  Main: [
+    //
```

Comment:
what is the purpose of this empty line comment?

Reply:
We're using `prettier` to format the files, since the array is short enough, it gets formatted to a single line. I added this comment as a workaround to force each item into a separate line.

Here's more information about it: https://github.com/prettier/prettier-vscode/issues/352

Reply:
That being said, the goal is to have PRs change the version independently with clearer diffs and reduce conflicts.

As an example, if we have a PR updating the latest Camel version and another one updating the latest LTS version, apart from the diff being a bit more complicated to read, we'll get conflicts since both PRs operating over the same line, with this change, the goal is to reduce the likeness for this to happen.